server/src/test/java/org/elasticsearch/cluster/coordination/FollowersCheckerTests.java

```diff
@@ -548,14 +546,16 @@ public String executor() {
         }
     }
 
-    private void testPreferMasterNodes() {
```

Comment:
I actually wasn't sure if this was "muted" on purpose, if it was maybe an @AwaitsFix annotation would be better. If not I hope the few lines I had to add to make the test pass locally are okay.